src/@next/components/atoms/RichTextEditorContent/RichTextEditorContent.tsx

```diff
@@ -45,12 +55,19 @@ export const RichTextEditorContent: React.FC<RichTextEditorContentProps> = ({
 
   React.useEffect(() => {
     if (data && editorContainer.current) {
-      editor.current = new EditorJS({
-        data,
-        holder: editorContainer.current,
-        readOnly: true,
-        tools,
-      });
+      (async () => {
+        const Editor: typeof EditorJS = await require("@editorjs/editorjs"); // eslint-disable-line  global-require
+
+        editor.current = new Editor({
+          data,
+          holder: editorContainer.current!,
+          // FIXME:
+          // Causes Uncaught (in promise) TypeError: Cannot read property 'deactivate' of null
```

Comment:
We are waiting for releasing bugfix for that https://github.com/codex-team/editor.js/issues/1380